providers/desktop_background.rb

```diff
@@ -11,9 +11,10 @@
 
 action :setup do
   begin
-    package "dconf-tools" do
```

Comment:
If you check all the commented lines of code, "dconf-tools" is not used here. To change the desktop background, the "gsettings.rb" provider is used. This provider uses "gsettings" command (instead of "dconf" command). And "gsettings" command belongs to "libglib2.0-bin" package.

So, since all that lines of code were commented, I commented also the package installation.